octobot_commons/profiles/profile_data.py

```diff
@@ -35,15 +35,18 @@ class ProfileDetailsData(octobot_commons.dataclasses.FlexibleDataclass):
 class CryptoCurrencyData(octobot_commons.dataclasses.FlexibleDataclass):
     trading_pairs: list[str]
     name: typing.Union[str, None] = None
+    leverage: typing.Union[float, None] = None
```

Comment:
this is to set leverage for a specific pair

Reply:
(same as the other comment)